Tests/test_PDB_parse_pdb_header.py

```diff
@@ -170,6 +172,24 @@ def test_parse_pdb_with_remark_99(self):
         self.assertEqual(header["astral"]["Region"], "a:")
         self.assertEqual(header["astral"]["ASTRAL-SPACI"], "0.72")
 
+    def test_parse_pdb_with_remark_350_biomoltrans(self):
+        """Tests that parse_pdb_header now can identify some REMARK 350 entries."""
+        header = parse_pdb_header("PDB/2XHE.pdb")
+        self.assertEqual(
+            header["biomoltrans"],
+            {
+                "1": [
+                    ["A", "B"],
+                    "  1.000000  0.000000  0.000000        0.00000            \n",
+                    "  0.000000  1.000000  0.000000        0.00000            \n",
+                    "  0.000000  0.000000  1.000000        0.00000            \n",
+                ]
+            },
+        )
+
+        header = parse_pdb_header("PDB/1LCD.pdb")
```

Comment:
I would separate these two tests, one to check for 2XHE and another to check that 1LCD does not have the remark.

Reply:
good point

Reply:
It may well be that the second test doesn't pass anymore. I need to check what the behaviour is now following these changes.

Reply:
> It may well be that the second test doesn't pass anymore. I need to check what the behaviour is now following these changes.

ok, this is still like that